core-rust/core-api-server/src/core_api/server.rs

```diff
@@ -93,8 +92,6 @@ pub async fn create_server<F>(
 {
     let core_api_state = CoreApiState { state_manager };
 
-    // TODO - Change to remove the Tower RequestBodyLimitLayer middleware and use DefaultBodyLimit::max
```

Comment:
Can we remove the tower dependency now? Or is it still used elsewhere?
Also could we update LARGE_REQUEST_MAX_BYTES now? 👍  I imagine it can be more like 3 MB now that the max transaction size is about 1MB?

Reply:
Done.